<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lathe</class>
 <widget class="QMainWindow" name="lathe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lathe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>831</width>
      <height>791</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="finalButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>finalmodel</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>770</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="followButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>follow</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modelButton">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>general</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yplus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>y+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yminus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>y-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cminus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>c-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aplus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>a+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aminus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>450</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>a-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cplus">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>c+</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusetting">
    <property name="title">
     <string>setting</string>
    </property>
    <addaction name="actionmodelfile"/>
    <addaction name="actioncodefile"/>
    <addaction name="actionoutput"/>
    <addaction name="actionknife"/>
   </widget>
   <addaction name="menusetting"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionstart"/>
   <addaction name="separator"/>
   <addaction name="actionpause"/>
   <addaction name="separator"/>
   <addaction name="actionfinalmodel"/>
   <addaction name="separator"/>
   <addaction name="actionoutput_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionmodelfile">
   <property name="text">
    <string>modelfile</string>
   </property>
  </action>
  <action name="actioncodefile">
   <property name="text">
    <string>codefile</string>
   </property>
  </action>
  <action name="actionoutput">
   <property name="text">
    <string>output</string>
   </property>
  </action>
  <action name="actionknife">
   <property name="text">
    <string>knife</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>start.png</normaloff>start.png</iconset>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>pause.png</normaloff>
     <normalon>pause.png</normalon>pause.png</iconset>
   </property>
   <property name="text">
    <string>pause</string>
   </property>
  </action>
  <action name="actionfinalmodel">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>end.png</normaloff>end.png</iconset>
   </property>
   <property name="text">
    <string>finalmodel</string>
   </property>
  </action>
  <action name="actionoutput_2">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>out.png</normaloff>out.png</iconset>
   </property>
   <property name="text">
    <string>output</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lathe.qrc"/>
 </resources>
 <connections/>
</ui>
